packages/components/src/box-control/utils.js

```diff
@@ -25,10 +25,10 @@ export const LABELS = {
 };
 
 export const DEFAULT_VALUES = {
-	top: null,
-	right: null,
-	bottom: null,
-	left: null,
+	top: undefined,
+	right: undefined,
+	bottom: undefined,
+	left: undefined,
```

Comment:
Since these values are passed to `UnitControl`'s `value`, `undefined` is a more correct value here